source/common/protobuf/message_validator_impl.cc

```diff
@@ -10,6 +12,28 @@
 namespace Envoy {
 namespace ProtobufMessage {
 
+void WarningValidationVisitorImpl::setCounter(Stats::Counter& counter) {
+  ASSERT(counter_ == nullptr, "");
```

Comment:
nit: the "" is redundant and you don't need it. Feel free to fix in a follow up if you come back in here or ignore.